src/vex.ts

```diff
@@ -1,59 +1,55 @@
 // [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
 //
-// ## Description
-// This file implements utility methods used by the rest of the VexFlow
-// codebase.
-//
+// Utility methods used by the rest of the VexFlow codebase.
 
 import { Flow } from './flow';
 import { log, RuntimeError } from './util';
 
 export const Vex = {
   Flow: Flow,
 
-  forEach:
-    // eslint-disable-next-line
-  (a: any[], fn: any) => {
-      for (let i = 0; i < a.length; i++) {
-        fn(a[i], i);
-      }
-    },
+  // eslint-disable-next-line
```

Comment:
This file was all little formatting fixes. I can't help it sometimes.... :-D

Reply:
I also do it. No change requied.